tests/conftest.py

```diff
@@ -0,0 +1,309 @@
+# -*- coding: utf-8 -*-
+
+# ======================================================================================================================
+# Imports
+# ======================================================================================================================
+import pytest
+
+
+# ======================================================================================================================
+# Fixtures
+# ======================================================================================================================
+@pytest.fixture(scope='session')
+def single_passing_xml(tmpdir_factory):
+    """JUnitXML sample representing a single passing test."""
+
+    filename = tmpdir_factory.mktemp('data').join('single_passing.xml').strpath
+    junit_xml = \
+    """<?xml version="1.0" encoding="utf-8"?>
+    <testsuite errors="0" failures="0" name="pytest" skips="0" tests="5" time="1.664">
+        <properties>
+            <property name="JENKINS_CONSOLE_LOG_URL" value="Unknown"/>
+            <property name="SCENARIO" value="Unknown"/>
+            <property name="ACTION" value="Unknown"/>
+            <property name="IMAGE" value="Unknown"/>
+            <property name="OS_ARTIFACT_SHA" value="Unknown"/>
+            <property name="PYTHON_ARTIFACT_SHA" value="Unknown"/>
+            <property name="APT_ARTIFACT_SHA" value="Unknown"/>
+            <property name="GIT_REPO" value="Unknown"/>
+            <property name="GIT_BRANCH" value="Unknown"/>
+        </properties>
+        <testcase classname="tests.test_default" file="tests/test_default.py" line="8" name="test_pass[ansible://localhost]" time="0.00372695922852"/>
+    </testsuite>
+    """
+
+    with open(filename, 'w') as f:
```

Comment:
Will the decorator `@pytest.fixture(scope='session')` clean up this file once its done with it?